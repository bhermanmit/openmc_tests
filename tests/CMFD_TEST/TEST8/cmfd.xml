<?xml version="1.0"?>
<cmfd>

  <mesh>
    <origin> -15 -1 -1 </origin>
    <dimension> 30 1 1 </dimension>
    <width> 1.0 2.0 2.0 </width>
    <albedo> 0.0 0.0 1.0 1.0 1.0 1.0 </albedo>
<!--    <map> 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 </map> -->

    <map> 1 1 1 1 1 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 1 1 1 1 1 </map>
  </mesh>

</cmfd>
