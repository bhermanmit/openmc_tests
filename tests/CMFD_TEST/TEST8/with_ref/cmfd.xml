<?xml version="1.0"?>
<cmfd>

  <mesh>
    <origin> -15 -1 -1 </origin>
    <dimension> 30 1 1 </dimension>
    <width> 1.0 2.0 2.0 </width>
    <albedo> 0.0 0.0 1.0 1.0 1.0 1.0 </albedo>
  </mesh>

</cmfd>
