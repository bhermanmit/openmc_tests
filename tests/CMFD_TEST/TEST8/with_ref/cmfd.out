MESH (1. 1, 1)  GROUP 1           -9.55778E-16
MESH (2. 1, 1)  GROUP 1           3.32769E-16
MESH (3. 1, 1)  GROUP 1           8.21825E-16
MESH (4. 1, 1)  GROUP 1           -2.27823E-15
MESH (5. 1, 1)  GROUP 1           -3.54534E-16
MESH (6. 1, 1)  GROUP 1           -2.34142E-06
MESH (7. 1, 1)  GROUP 1           4.77904E-05
MESH (8. 1, 1)  GROUP 1           1.97958E-05
MESH (9. 1, 1)  GROUP 1           1.01404E-04
MESH (10. 1, 1)  GROUP 1          -1.12975E-04
MESH (11. 1, 1)  GROUP 1          1.23142E-04
MESH (12. 1, 1)  GROUP 1          -5.46020E-06
MESH (13. 1, 1)  GROUP 1          1.13549E-05
MESH (14. 1, 1)  GROUP 1          2.68231E-05
MESH (15. 1, 1)  GROUP 1          8.84345E-05
MESH (16. 1, 1)  GROUP 1          -3.68613E-05
MESH (17. 1, 1)  GROUP 1          -3.24334E-06
MESH (18. 1, 1)  GROUP 1          1.28618E-05
MESH (19. 1, 1)  GROUP 1          7.08386E-05
MESH (20. 1, 1)  GROUP 1          1.53751E-04
MESH (21. 1, 1)  GROUP 1          2.66623E-05
MESH (22. 1, 1)  GROUP 1          -4.83628E-05
MESH (23. 1, 1)  GROUP 1          -1.18597E-04
MESH (24. 1, 1)  GROUP 1          3.29678E-05
MESH (25. 1, 1)  GROUP 1          1.98921E-04
MESH (26. 1, 1)  GROUP 1          9.81203E-18
MESH (27. 1, 1)  GROUP 1          -6.21573E-16
MESH (28. 1, 1)  GROUP 1          9.66349E-16
MESH (29. 1, 1)  GROUP 1          -9.99634E-16
MESH (30. 1, 1)  GROUP 1          -3.83076E-16
